packages/@aws-cdk/toolkit-lib/lib/api/io/toolkit-action.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * The current action being performed by the CLI. 'none' represents the absence of an action.
+ * The current action being performed by the Toolkit or CLI.
```

Comment:
Technically the Toolkit is both.
```suggestion
 * The current action being performed by the Toolkit.
```